plugins/outputs/datadog/README.md

```diff
@@ -18,6 +18,9 @@ This plugin writes to the [Datadog Metrics API][metrics] and requires an
 
   ## Set http_proxy (telegraf uses the system wide proxy settings if it isn't set)
   # http_proxy_url = "http://localhost:8888"
+
+  ## Whether to compress the HTTP request body
+  # compress = true
```

Comment:
Is it possible for Datadog to receive other compression formats? Maybe instead of a boolean we would want a string here to support other compression formats in the future (e.g. gzip) this would help with backwards compatibility as well if Datadog will support a different compression type in the future.

```suggestion
  ## Override the default (zlib) compression used to send data.
  ## Supports: "zlib", "none"
  # compression = "none"
```

Reply:
At the moment no, they only support zlib. But can certainly update it to your suggestion if that is the desired route.

Reply:
Just in case they do add other options, I think this would be the better route if you don't mind updating it. Thank you for your work!

Reply:
Updated!